models/stg_hubspot__contact.yml

```diff
@@ -4,7 +4,7 @@ models:
   - name: stg_hubspot__contact_list_member
     description: Each record represents a 'link' between a contact and a contact list.
     columns:
-      - name: _fivetran_deleted
+      - name: is_contact_list_member_deleted
         description: '{{ doc("_fivetran_deleted") }}'
```

Comment:
should all of these be `'{{ doc("is_deleted") }}'`?  or we should change the definition of `_fivetran_deleted` in the docs.md, as it's a boolean and not a timestamp

Reply:
(this applies across the board, not just for contact_list_member)

Reply:
Oddly, some tables are using `_fivetran_deleted` vs `is_deleted`. I could see pros and cons for both approaches (1. using just `is_deleted` vs 2. using `_fivetran_deleted` and `is_deleted` respectively). I think I'm personally leaning toward keeping them separate, so option 2. Just so that we can align with the actual source data. Do you have any strong thoughts here?

Reply:
I will also update _fivetran_deleted to reflect the proper definition!